### PR TITLE
Some README edits

### PR DESCRIPTION
**Pull request by dstettler**
A somewhat subjective edit to the main README that I felt would be useful as I was setting up my dev environment.
Changes:

- A table of contents linking to the different sections
- Some typos/grammar things changed (Submitting Issue -> Submitting Issues, etc.)
- A short section about setting up a build environment. I didn't see anything in the README about this already, so when I was first setting up, I skimmed the Makefile and managed to miss the nightly Rust requirement, and that the build task didn't install the Pip package, and was promptly confused by errors for a bit. Explicitly mentioning these in the README could be useful.

### CODE REVIEW

**Comment by merwok** [comment]
github automatically generates table of contents for markdown files

**Comment by dstettler** [comment]
Ah, that is true, I can remove the TOC from the branch, I will admit I primarily use Bitbucket and forgot that GitHub did that. Would anyone be fundamentally opposed to my other changes? If so, I could just close the PR.

File: README.md [hunk 1]
```diff
@@ -558,15 +558,25 @@ If you are familiar with your skills, try to create amazing works with [this](py
 
 ## How to Contribute
 
-### Submitting Issue
+### Building Overview
+
+Pyxel is built primarily using Rust, and installed via Pip. Ensure you have the *nightly* version of Rust installed with `rustup`, or `make` will generate build errors.
+
+The `make clean build` command will not install the build, so ensure you run ``pip3 install --force-reinstall `ls -rt ./dist/*.whl | tail -n 1` `` followed by `python3 -m unittest discover ./crates/pyxel-extension/tests` to install it.
```
Comment (merwok) [inline README.md:565]: IMO the pip command is too complex, maybe a new makefile target is needed.
(note that I am not a project maintainer)

I would remove the mention of the tests, that’s not really useful for game devs.

**Comment by kitao** [comment]
Thank you for your contribution.
Pyxel's REAMDE has some policies. Specifically
- Include only information for beginners and keep the content as simple as possible
- No special treatment for English, and when updating, update for 10 languages at the same time
The way to build Pyxel  is too complicated for beginners and also it changes often. So I have no intention to describe it in the READMEs.

**Comment by dstettler** [comment]
Thanks so much for responding and the thoughts! I was hoping to try to help with getting possible contributors up to speed, but these are all really fair points, and I suppose I didn't really consider the target audience of the main README.

I've removed all but my grammatical changes, but they're rather nitpicky and subjective, so I could close the PR instead if that would be better.

**Review by merwok — APPROVED** [review]
small but nice fixes for english!

**Comment by kitao** [comment]
Thank you for your fixing.
Word and grammar corrections are welcome because I'm not a native English speaker.
Please let me know if you notice anything about other languages as well.

**Pull request merged**